docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,309 @@
+
+### Choosing user attributes for sandboxing
+
+When [adding a new user attribute](../people-and-groups/managing.md#adding-a-user-attribute), you'll set up a key-value pair for each person.
+
+The **user attribute key** is used to look up the **user attribute value** for a specific person. User attribute keys can be mapped to parameters in Metabase, such as [SQL parameters](../questions/native-editor/sql-parameters.md). For an example, see [How row restriction works in an advanced sandbox](#how-row-restriction-works-in-an-advanced-sandbox).
+
+The **user attribute value** must be an exact, case-sensitive match for the filter value of a sandboxed table. For example, if you're creating a [row-restricted sandbox](#row-restricted-sandboxes) on the Accounts table with the filter `Plan = "Basic"`, make sure that you enter "Basic" as the user attribute value. If you set the user attribute value to lowercase "basic" (a value which doesn't exist in the Plan column of the Accounts table), the sandboxed person will get an empty result instead of the sandboxed table.
+
+## Creating a row-restricted sandbox
+
+1. Make sure to do the [prerequisites for row-restricted sandboxes](#prerequisites-for-row-restricted-sandboxes) first.
+2. Go to **Admin settings** > **Permissions**.
+3. Select the database and table that you want to sandbox.
+4. Find the group that you want to put in the sandbox.
+5. Click on the dropdown under **Data access** for that group.
+6. Select "Sandboxed".
+7. Click the dropdown under **Column** and enter the filter name for your table, such as "Plan".
+8. Click the dropdown under **User attribute** and enter the user attribute **key**, such as "User's Plan".
+
+> If you have saved SQL questions that use sandboxed data, make sure to move all of those questions to admin-only collections. For more info, see [Permissions conflicts: saved SQL questions](#saved-sql-questions).
+
+For a tutorial, check out [Data sandboxing: setting row-level permissions](https://www.metabase.com/learn/permissions/data-sandboxing-row-permissions).
+
+## Prerequisites for advanced sandboxes
+
+- A [group](../people-and-groups/managing.md#groups) of people to be added to the advanced data sandbox.
+- An admin-only [collection](../exploration-and-organization/collections.md) ([collection permissions](../permissions/collections.md) set to **No access** for all groups except Administrators).
+- A [saved SQL question](../people-and-groups/) with the rows and columns to be displayed to the people in the advanced sandbox, stored in the admin-only collection.
+- Optional: [User attributes](../people-and-groups/managing.md#adding-a-user-attribute) for each of the people in the group.
+
+To display different filtered versions of the saved question for each person in an advanced data sandbox, set up [user attributes](../people-and-groups/managing.md#adding-user-attributes) as well.
+
+### Designing a saved question to be displayed in an advanced sandbox
```

Comment:
```suggestion
### Create a SQL question that Metabase can use to create a custom view of the table you want to sandbox
```

Reply:
I have been writing "displays X" instead of "view of X" to avoid confusing a sandboxed table with a materialized view of that table.

Though I wasn't sure if Metabase _was_ creating a materialized view for a sandboxed table. But I just looked it up, and views require CREATE privileges. Luckily, we worked on privileges recently, so I'm pretty sure the only feature that requires CREATE is model caching.